.vscode/settings.json

```diff
@@ -24,8 +24,7 @@
     "CONTRIBUTING": "markdown"
   },
   "yaml.schemas": {
-    // "http://json-schema.org/draft/2019-09/schema#": "./values-schema.yaml",
-    // "http://json-schema.org/draft/2019-09/schema#": ".vscode/values-schema.yaml",
+    "http://json-schema.org/draft-07/schema#": "values-schema.yaml",
```

Comment:
I disabled this as it was also applying the same schema suddenly to the values. Can you confirm your change is working as intended and not breaking values validation?

Reply:
I don't understand, you mean that it would evaluate any `.yaml` file with the `json-schema draft 07`? I didn't have that experience on my computer to confirm.